app/actions/websocket/users.ts

```diff
@@ -203,3 +207,19 @@ export function handleUserUpdatedEvent(msg: WebSocketMessage) {
         return {data: true};
     };
 }
+
+export function handleUserSessionRevokedEvent() {
+    return (dispatch: DispatchFunc, getState: GetStateFunc): ActionResult => {
+        dispatch(logout(true));
+
+        const locale = getCurrentLocale(getState()) || General.DEFAULT_LOCALE;
+        const translations = getTranslations(locale) as {[key: string]: string};
+
+        Alert.alert(
+            translations['mobile.session_revoked.title'],
+            translations['mobile.session_revoked.message'],
+        );
```

Comment:
Feels weird that this is the right way to do it. Also, what happens is that language in particular does not have that string? Will it correctly default to English, or will it show an empty string (or worse, `undefined` as a string)?

Reply:
Just to help out with this: 

* It feels weird but if I'm not mistaken, that would be the right way to go if we don't the access the `useIntl` context - when you're not in a component.  @jfrerich maybe you can evaluate this solution for #6449 
* In the event an issue arise due to a missing string, it would need to default to the primary locale.  On the Gekidou branch, I see that there's a fix for it.  See `PRIMARY_LOCALE` at the end of the line https://github.com/mattermost/mattermost-mobile/blob/f876dc595152d4c677cfa784160d7c6a572f7ce6/app/i18n/index.ts#L180  We would need to add it into this PR.  But still then, let's test if it properly works :)